### PR TITLE
bug 1720178 updating post-install CSR approval

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1720178

@deads2k, PTAL?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@deads2k, PTAL?

**Comment by deads2k** [comment]
I think Seth is closer to this.  

/assign @sjenning

**Comment by kalexand-rh** [comment]
@sjenning, will you PTAL?

File: modules/installation-approve-csrs.adoc [hunk 1]
```diff
@@ -64,7 +64,8 @@ of adding the machines to the cluster. If you do not approve them within an
 hour, the certificates will rotate, and more than two certificates will be
 present for each node. You must approve all of these certificates. After you
 approve the initial CSRs, the subsequent CSRs are automatically approved by the
-cluster `kube-controller-manger`.
+cluster `kube-controller-manger`. You must implement a method of automatically
```
Comment (DanyC97) [inline modules/installation-approve-csrs.adoc]: typo?  `manger` -> `manager`

**Comment by sjenning** [comment]
LGTM

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL at this small change.

**Comment by mburke5678** [comment]
@kalexand-rh LGTM

**Comment by kalexand-rh** [comment]
Thanks everyone! Jianlin approved this change on the source bug, so I'm going to merge.

**Pull request merged**